README.md
# Mountain Legacy Project: Explorer Application
(c) 2021 Runtime Software Development Inc.

## To be added soon.


CHANGELOG.md
# Changelog
All notable changes to this project will be documented in this file.

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [Unreleased]

## [1.0.0] - 2017-06-20
### Added
- New visual identity by [@tylerfortune8](https://github.com/tylerfortune8).
- Version navigation.
- Links to latest released version in previous versions.
- "Why keep a changelog?" section.
- "Who needs a changelog?" section.
- "How do I make a changelog?" section.
- "Frequently Asked Questions" section.
- New "Guiding Principles" sub-section to "How do I make a changelog?".
- Simplified and Traditional Chinese translations from [@tianshuo](https://github.com/tianshuo).
- German translation from [@mpbzh](https://github.com/mpbzh) & [@Art4](https://github.com/Art4).
- Italian translation from [@azkidenz](https://github.com/azkidenz).
- Swedish translation from [@magol](https://github.com/magol).
- Turkish translation from [@karalamalar](https://github.com/karalamalar).
- French translation from [@zapashcanon](https://github.com/zapashcanon).
- Brazilian Portugese translation from [@Webysther](https://github.com/Webysther).
- Polish translation from [@amielucha](https://github.com/amielucha) & [@m-aciek](https://github.com/m-aciek).
- Russian translation from [@aishek](https://github.com/aishek).
- Czech translation from [@h4vry](https://github.com/h4vry).
- Slovak translation from [@jkostolansky](https://github.com/jkostolansky).
- Korean translation from [@pierceh89](https://github.com/pierceh89).
- Croatian translation from [@porx](https://github.com/porx).
- Persian translation from [@Hameds](https://github.com/Hameds).
- Ukrainian translation from [@osadchyi-s](https://github.com/osadchyi-s).

### Changed
- Start using "changelog" over "change log" since it's the common usage.
- Start versioning based on the current English version at 0.3.0 to help
translation authors keep things up-to-date.
- Rewrite "What makes unicorns cry?" section.
- Rewrite "Ignoring Deprecations" sub-section to clarify the ideal
  scenario.
- Improve "Commit log diffs" sub-section to further argument against
  them.
- Merge "Why can’t people just use a git log diff?" with "Commit log
  diffs"
- Fix typos in Simplified Chinese and Traditional Chinese translations.
- Fix typos in Brazilian Portuguese translation.
- Fix typos in Turkish translation.
- Fix typos in Czech translation.
- Fix typos in Swedish translation.
- Improve phrasing in French translation.
- Fix phrasing and spelling in German translation.

### Removed
- Section about "changelog" vs "CHANGELOG".

## [0.3.0] - 2015-12-03
### Added
- RU translation from [@aishek](https://github.com/aishek).
- pt-BR translation from [@tallesl](https://github.com/tallesl).
- es-ES translation from [@ZeliosAriex](https://github.com/ZeliosAriex).

## [0.2.0] - 2015-10-06
### Changed
- Remove exclusionary mentions of "open source" since this project can
benefit both "open" and "closed" source projects equally.

## [0.1.0] - 2015-10-06
### Added
- Answer "Should you ever rewrite a change log?".

### Changed
- Improve argument against commit logs.
- Start following [SemVer](https://semver.org) properly.

## [0.0.8] - 2015-02-17
### Changed
- Update year to match in every README example.
- Reluctantly stop making fun of Brits only, since most of the world
  writes dates in a strange way.

### Fixed
- Fix typos in recent README changes.
- Update outdated unreleased diff link.

## [0.0.7] - 2015-02-16
### Added
- Link, and make it obvious that date format is ISO 8601.

### Changed
- Clarified the section on "Is there a standard change log format?".

### Fixed
- Fix Markdown links to tag comparison URL with footnote-style links.

## [0.0.6] - 2014-12-12
### Added
- README section on "yanked" releases.

## [0.0.5] - 2014-08-09
### Added
- Markdown links to version tags on release headings.
- Unreleased section to gather unreleased changes and encourage note
keeping prior to releases.

## [0.0.4] - 2014-08-09
### Added
- Better explanation of the difference between the file ("CHANGELOG")
and its function "the change log".

### Changed
- Refer to a "change log" instead of a "CHANGELOG" throughout the site
to differentiate between the file and the purpose of the file — the
logging of changes.

### Removed
- Remove empty sections from CHANGELOG, they occupy too much space and
create too much noise in the file. People will have to assume that the
missing sections were intentionally left out because they contained no
notable changes.

## [0.0.3] - 2014-08-09
### Added
- "Why should I care?" section mentioning The Changelog podcast.

## [0.0.2] - 2014-07-10
### Added
- Explanation of the recommended reverse chronological release ordering.

## [0.0.1] - 2014-05-31
### Added
- This CHANGELOG file to hopefully serve as an evolving example of a
  standardized open source project CHANGELOG.
- CNAME file to enable GitHub Pages custom domain
- README now contains answers to common questions about CHANGELOGs
- Good examples and basic guidelines, including proper date formatting.
- Counter-examples: "What makes unicorns cry?"

[Unreleased]: https://github.com/olivierlacan/keep-a-changelog/compare/v1.0.0...HEAD
[1.0.0]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.3.0...v1.0.0
[0.3.0]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.2.0...v0.3.0
[0.2.0]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.1.0...v0.2.0
[0.1.0]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.0.8...v0.1.0
[0.0.8]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.0.7...v0.0.8
[0.0.7]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.0.6...v0.0.7
[0.0.6]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.0.5...v0.0.6
[0.0.5]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.0.4...v0.0.5

labels.js
'use strict';

/**
 * Rendering settings
 * @private
 */

export default {
    main: {
        projectName: "Mountain Legacy Project",
        appName: "Explorer",
        title: "Welcome to the Mountain Legacy Project Explorer"
    },
    projects: {
        id: {
            label: "ID"
        },
        name: {
            label: 'Project Name'
        },
        description: {
            label: 'Project Description'
        },
    },
    surveyors: {
        id: {
            label: "ID"
        },
        given_names: 'Given Names',
        last_name: 'Last Name',
        short_name: 'TEST',
        affiliation: "Affiliation"
    },
    surveys: {
        id: {
            label: "ID"
        },
        owner_id: 16,
        name: 'Some Name',
        historical_map_sheet: 'Historical Map Sheet',
    },
    surveySeasons: {
        id: {
            label: "ID"
        },
        owner_id: 16,
        year: 1933,
        geographic_coverage: 'TEST',
        record_id: 0,
        jurisdiction: 'TEST',
        affiliation: 'TEST',
        archive: 'TEST',
        collection: 'TEST',
        location: 'TEST',
        sources: 'TEST',
        notes: 'TEST'
    },
    stations: {
        id: {
            label: "ID"
        },
        owner_id: 40,
        owner_type: 'survey_seasons',
        name: 'TEST',
        lat: 100.1,
        long: 100.1,
        elevation: 100.1,
        nts_sheet: 'TEST',
        published: false
    },
    visits: {
        id: {
            label: "ID"
        },
        owner_id: 61,
        date: '2005-08-19',
        start_time: '14:00:00',
        finish_time: '17:00:00',
        pilot: 'TEST',
        rw_call_sign: 'TEXT',
        visit_narrative: 'TEXT',
        illustration: false,
        weather_narrative: 'TEXT',
        weather_temp: 14,
        weather_ws: 25,
        weather_gs: 34,
        weather_pressure: 101,
        weather_rh: 15,
        weather_wb: 22
    },
    historicVisits: {
        id: {
            label: "ID"
        },
        owner_id: 9,
        date: '1927-01-01',
        comments: 'TEXT'
    },
    locations: {
        id: {
            label: "ID"
        },
        owner_id: 23,
        location_narrative: 'TEXT',
        location_identity: 'TEXT',
        lat: 100.1,
        long: 100.1,
        elevation: 100.1,
        legacy_photos_start: 5,
        legacy_photos_end: 8,
        published: true
    },
    historicCaptures: {
        id: {
            label: "ID"
        },
        owner_id: 64,
        owner_type: 'survey_seasons',
        plate_id:'529',
        fn_photo_reference: 'TEXT',
        f_stop: 555,
        shutter_speed: 34,
        focal_length: 12,
        capture_datetime: '2014-07-09 16:49:00.572006',
        camera_id: 6,
        lens_id: null,
        digitization_location: 'LAC',
        digitization_datetime: '2014-07-09 16:49:00.572006',
        lac_ecopy: 'IDENTIFIER',
        lac_wo: 'IDENTIFIER',
        lac_collection: 'IDENTIFIER',
        lac_box: 'IDENTIFIER',
        lac_catalogue: 'IDENTIFIER',
        condition: 'DESCRIPTION',
        comments: 'TEXT'
    },
    captures:{
        id: {
            label: "ID"
        },
        owner_id: 1051,
        owner_type: 'locations',
        fn_photo_reference: 'IDENTIFIER',
        f_stop: 55,
        shutter_speed: 55,
        focal_length: 56,
        capture_datetime: '2014-07-09 16:49:00.572006',
        camera_id: 6,
        lens_id: null,
        lat: 100.1,
        long: 100.1,
        elevation: 100.1,
        azimuth: 300,
        comments: 'TEXT',
        alternate: true
    },
    captureImages:{
        id: {
            label: "ID"
        },
        owner_id: 3897,
        owner_type: 'historic_captures',
        hash_key: 'IDENTIFIER',
        image: 'IDENTIFIER',
        file_size: 5566,
        x_dim: 1200,
        y_dim: 1600,
        image_state: 'MASTER',
        image_remote: 'IDENTIFIER',
        image_secure_token: 'IDENTIFIER',
        comments: 'TEXT',
        image_tmp: 'IDENTIFIER',
        image_remote_processing: false
    },
    images: {
        id: {
            label: "ID"
        },
        owner_id: 483,
        owner_type: 'locations',
        hash_key: 'IDENTIFIER',
        image: 'LC2_DSCF0342_011f9804-7099-11e2-a556-c82a14fffed2.JPG',
        file_size: 89089504,
        x_dim: 1200,
        y_dim: 1600,
        bit_depth: 8,
        image_remote: 'IDENTIFIER',
        image_secure_token: 'IDENTIFIER',
        comments: 'TEXT',
        lat: 100.1,
        long: 100.1,
        f_stop: 55,
        shutter_speed: 55,
        focal_length: 56,
        iso: 50,
        capture_datetime: '2014-07-09 16:49:00.572006',
        camera_id: 6,
        lens_id: null,
        type: 'location',
        image_tmp: 'IDENTIFIER',
        image_remote_processing: false
    },
    cameras: {
        id: {
            label: "ID"
        },
        make: 'TEXT',
        model: 'TEXT',
        unit: 'TEXT',
        format: 'TEXT'
    },
    glass_plate_listings: {
        id: {
            label: "ID"
        },
        owner_id: 93,
        container: 'TEXT',
        plates: 'TEXT',
        notes: 'TEXT'
    },
    lens: {
        id: {
            label: "ID"
        },
        brand: 'TEXT',
        focal_length: 4,
        max_aperture: 1.11
    },
    maps: {
        id: {
            label: "ID"
        },
        owner_id: 36,
        nts_map: 'TEXT',
        historic_map: 'TEXT',
        links: 'TEXT'
    },
    metadata_files:{
        id: {
            label: "ID"
        },
        owner_id: 992,
        owner_type: 'stations',
        type: 'field_notes',
        filename: 'PATH/to/FILE'
    },
    participants: {
        id: {
            label: "ID"
        },
        last_name: 'TEXT',
        given_names: 'TEXT'
    },
    participant_groups: {
        id: {
            label: "ID"
        },
        owner_id: 23,
        participant_id: 38,
        group_type: 'photographers'
    }
}
